apps/client/src/app/components/admin-market-data/create-asset-profile-dialog/create-asset-profile-dialog.html

```diff
@@ -8,7 +8,10 @@ <h1 i18n mat-dialog-title>Create Asset Profile</h1>
   <div class="flex-grow-1 py-3" mat-dialog-content>
     <mat-form-field appearance="outline" class="w-100">
       <mat-label i18n>Name, symbol or ISIN</mat-label>
-      <gf-symbol-autocomplete formControlName="searchSymbol" />
+      <gf-symbol-autocomplete
+        formControlName="searchSymbol"
+        includeIndex="true"
```

Comment:
I usually use `[includeIndices]="true"` to make sure it's a `boolean` and not a `string`